panel/widgets/tables.py

```diff
@@ -260,77 +706,336 @@ def _process_param_change(self, msg):
             msg['editable'] = not msg.pop('disabled') and len(self.indexes) <= 1
         return super(DataFrame, self)._process_param_change(msg)
 
+    def _update_aggregators(self, model):
+        for g in model.grouping:
+            group = self._renamed_cols.get(g.getter, g.getter)
+            index = self.indexes[self.indexes.index(group)+1]
+            g.aggregators = self._get_aggregators(index)
+
+
+class Tabulator(BaseTable):
+    """
+    The Tabulator Pane wraps the [Tabulator](http://tabulator.info/)
+    table to provide an awesome interactive table.
+    """
+
+    frozen_columns = param.List(default=[], doc="""
+        List indicating the columns to freeze. The column(s) may be
+        selected by name or index.""")
+
+    frozen_rows = param.List(default=[], doc="""
+        List indicating the rows to freeze. If set the
+        first N rows will be frozen which prevents them from scrolling
+        out of frame, if set to a negative value last N rows will be
+        frozen.""")
```

Comment:
```suggestion
        List indicating the rows to freeze. If set, the
        first N rows will be frozen, which prevents them from scrolling
        out of frame; if set to a negative value the last N rows will be
        frozen.""")
```

Shouldn't these parameters that are shared with the DataFrame class be grouped into a base class so that the docstrings don't drift out of sync?

Reply:
They are quite different unfortunately, one accepts a single int this one accepts a list.